CustomDimensions.php

```diff
@@ -280,10 +281,13 @@ public function addConversionInformation(&$conversion, $visitInformation, Tracke
 
         foreach ($dimensions as $dimension) {
             $index = (int) $dimension['index'];
-            if ($dimension['scope'] === self::SCOPE_VISIT && in_array($index, $conversionIndexes)) {
+            if (in_array($dimension['scope'], array(self::SCOPE_ACTION, self::SCOPE_VISIT)) && in_array($index, $conversionIndexes)) {
                 $field = LogTable::buildCustomDimensionColumnName($dimension);
 
-                if (array_key_exists($field, $visitInformation)) {
+                if ($dimension['scope'] === self::SCOPE_ACTION && array_key_exists($field, $actionDimensions)) {
```

Comment:
Yep, looks like it. Not sure what best way would be to solve it then. I presume we would need to check if there is an action set, and if so, get values from the action. If no action is set, get them like you implemented directly from `getCustomDimensionsInScope`. Would that be possible?